### PR TITLE
Add sanic plugin

### PR DESCRIPTION
**Pull request by fuhuo**
- [x] Add a test case for the new plugin
- [x] Add a component id in [the main repo](https://github.com/apache/skywalking/blob/master/oap-server/server-bootstrap/src/main/resources/component-libraries.yml#L415)
- [x] Add a logo in [the UI repo](https://github.com/apache/skywalking-rocketbot-ui/tree/master/src/views/components/topology/assets)
- [x] Rebuild the `requirements.txt` by running `tools/env/build_requirements_(linux|windows).sh`

### CODE REVIEW

**Comment by kezhenxu94** [comment]
@fuhuo thanks for contributing, can you please fix the CI and check the item list in the?

**Comment by fuhuo** [comment]
ok, i'll try

**Comment by kezhenxu94** [comment]
> ok, i'll try

If you need any help, feel free to ping me

**Comment by kezhenxu94** [comment]
Please check this one by one before we can merge this PR, thanks, if you need help, feel free to ping me anytime

image

File: skywalking/plugins/sw_sanic.py
Comment (kezhenxu94) [inline skywalking/plugins/sw_sanic.py:47]: Please don't use a global variable to store the status code, in concurrent scenario, the status code may mess up in different requests.

You can use the same way as line 81 to get the current active span (nullablity check) and set the status code here

```python
entry_span = get_context().active_span()
```

**Review by kezhenxu94 — COMMENTED** [review]
Also, could you please also update the plugin list [here](https://github.com/apache/skywalking-python/blob/master/docs/Plugins.md), thanks very much

File: skywalking/plugins/sw_sanic.py
Comment (fuhuo) [reply skywalking/plugins/sw_sanic.py:47]: ok,thanks.i'll try to fix it later

**Review by kezhenxu94 — APPROVED** [review]
Thanks very much for your contributions and patience

**Pull request merged**
